commands/Moderation/role.js

```diff
@@ -0,0 +1,27 @@
+exports.run = async (client, msg, [user, role]) => {
+  if (!msg.guild.roles.exists("name", role)) return msg.reply("There is no role by that name in this server.");
```

Comment:
You can use Roles#find instead, so you don't retrieve the role by its name twice.